pythonosc/osc_message_builder.py

```diff
@@ -72,6 +75,8 @@ def add_arg(self, arg_value, arg_type=None):
         arg_type = self.ARG_TYPE_TRUE
       elif arg_value == False:
         arg_type = self.ARG_TYPE_FALSE
+      elif arg_value is None:
```

Comment:
please check below l.73 above for the float argument if it's equal to infinity, in which case set the right arg_type